tutorials/mixed_dimensional_grids.ipynb

```diff
@@ -21,7 +21,7 @@
     "For most simulation purposes, the final mixed-dimensional grid is all that is needed. Therefore, we start by showing a shortcut for obtaining a `MixedDimensionalGrid` given a set of fractures, a domain and mesh size parameters. All these will be described in more detail below.\n",
```

Comment:
<p>First, this introductory paragraph I liked a lot, it is easy to see how that relates to the roadmap that was partially indicated in the first tutorial.</p><p> </p><p>Regarding the edit, I am not sure we do anyone a favor by sending them into that particular folder.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1545/discussion/#comment-2585466215'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>